### DISCUSSION TITLE
Please add some porn website extensions

### DISCUSSION THREAD

**Discussion by Abubakar1009786** [aniyomiorg/aniyomi#532] [Ideas]
> your app is.good but if you add some porn website extensions your app will be best please add now please please please please please please please please please please please please please please please please please please please add these extension. And improve some speed

**Comment by Abubakar1009786** [comment]
Tell me people do you want it

**Reply by Tluanga-khawlhring** [reply]
Got to www.pornhub.com 😅

**Reply by Acion-don** [reply]
💀💀💀💀💀💀

**Comment by 12ohit** [comment]
Here you go you perverts https://github.com/jmir1/unofficial-aniyomi-extensions/tree/repo/apk

**Reply by subakuma** [reply]
Could you make one for JAV?

**Reply by Perpendiculr** [reply]
None of them working bruh u ruined the mood only twitch is working

**Comment by FrenchPrick** [comment]
Why don't you just get one of the millions different porn apps.

**Comment by FrenchPrick** [comment]
Um excuse me, I wasn't the one who opened this request in the first place.

**Comment by FrenchPrick** [comment]
Porn is bad for your mental health.

**Reply by durjoyhalder734-sudo** [reply]
It relax our mind when we sleep😂

**Comment by RisingTM** [comment]
I really have to agree with the guy that said porn is bad for your health ... bcs it is ...I am not one to judge what you do with your life ... but just a tip ... stop being down bad that you ask for a porn extension in an app for anime ... (it's really a player for anime, but you get the idea)

**Comment by durjoyhalder734-sudo** [comment]
Add more and more extention for😋

**Comment by jaydeep2005** [comment]
Do anyone have any problem with animetsu extension, it's not working, it has a stopped Fully, it's related to json, please developer.
